fenix/app/src/main/java/org/mozilla/fenix/utils/Settings.kt

```diff
@@ -1811,7 +1812,11 @@ class Settings(private val appContext: Context) : PreferencesHolder {
      */
     val enableShoppingExperience by booleanPreference(
         key = appContext.getPreferenceKey(R.string.pref_key_enable_shopping_experience),
-        default = FxNimbus.features.shoppingExperience.value().enabled,
+        default = FxNimbus.features.shoppingExperience.value().enabled.also {
+            ShoppingSettings.settingsNimbusDisabledShopping.set(!it)
+            ShoppingSettings.userHasOnboarded.set(false)
+            ShoppingSettings.settingsComponentOptedOut.set(false)
```

Comment:
These are set as part of metrics ping, I don't think we need to add them as events as well